plugins/toml.ts

```diff
@@ -0,0 +1,30 @@
+import toml from "../core/loaders/yaml.ts";
+import { merge } from "../core/utils.ts";
+
+import type { Site } from "../core.ts";
+
+export interface Options {
+  /** The list of extensions this plugin applies to */
+  extensions: string[] | {
+    pages: string[];
+    data: string[];
+  };
+}
+
+// Default options
+export const defaults: Options = {
+  extensions: [".toml"],
+};
+
+/** A plugin to add support for YAML files */
```

Comment:
Just a tiny thing: replace YAML with TOML in this comment

Reply:
> Just a tiny thing: replace YAML with TOML in this comment

Thanks, didn't notice. Fixed!